lxd/daemon.go

```diff
@@ -1841,29 +1840,14 @@ func (d *Daemon) setupRBACServer(rbacURL string, rbacKey string, rbacExpiry int6
 		return err
 	}
 
-	// Perform full sync when online
```

Comment:
This change and the resulting place where it gets moved to feel like they would benefit being a separate commit so its not lost in the mechanical changes.